src/System.Threading/ref/System.Threading.cs

```diff
@@ -32,7 +32,8 @@ public partial struct AsyncFlowControl : System.IDisposable
     }
     public readonly partial struct AsyncLocalValueChangedArgs<T>
     {
-        private readonly T _dummy;
+        private readonly T _PreviousValue_k__BackingField;
```

Comment:
It looks like we don't have that many other instances of such names within the repo. I guess this is how the underlying field name gets generated as for an auto-prop.
https://github.com/dotnet/corefx/blob/b2097cbdcb26f7f317252334ddcce101a20b7f3d/src/Common/src/CoreLib/System/Threading/AsyncLocal.cs#L93

The other instances are in Reflection.Metadata:
https://github.com/dotnet/corefx/blob/b2097cbdcb26f7f317252334ddcce101a20b7f3d/src/System.Reflection.Metadata/src/System/Reflection/Metadata/Signatures/CustomAttributeNamedArgument.cs#L11
https://github.com/dotnet/corefx/blob/b2097cbdcb26f7f317252334ddcce101a20b7f3d/src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs#L497

Reply:
Yes, I think this is the auto-generated backing field. I think it is fine right? To be deterministic when running GenAPI and also since these are private, it doesn't really affect surface area.